bin/node-pg-migrate

```diff
@@ -221,6 +212,32 @@ if (argv.help || argv._.length === 0) {
   process.exit(1)
 }
 
+/* Load env before accessing process.env */
+const envPath = argv[envPathArg]
+
+try {
+  // Create default dotenv config
+  const dotenvConfig = { silent: true }
+
+  // If the path has been configured, add it to the config, otherwise don't change the default dotenv path
+  if (envPath) {
+    dotenvConfig.path = envPath
+  }
+
+  const myEnv = require('dotenv').config(dotenvConfig) // eslint-disable-line global-require,import/no-extraneous-dependencies
+  try {
+    require('dotenv-expand')(myEnv) // eslint-disable-line global-require,import/no-extraneous-dependencies
+  } catch (err) {
```

Comment:
**thought (if-minor):** @goce-cz We have an inner and an outer try-catch here :thinking:  
Should we remove the inner one and just let the outer handle everything?

Reply:
Yes, since I touched this code, I will refactor this.

Reply:
Thanks. Hopefully these try/catches will be gone for good soon: #752